vars/uploadArtifacts.groovy

```diff
@@ -11,10 +15,15 @@ void call(Map args = [:]) {
         s3Upload(file: 'dist', bucket: "${ARTIFACT_BUCKET_NAME}", path: "${artifactPath}/dist")
     }
 
+    withAWS(role: "${ARTIFACT_PROMOTION_ROLE_NAME}", roleAccount: "${AWS_ACCOUNT_PUBLIC_REAL}", duration: 900, roleSessionName: 'jenkins-session') {
+        s3Upload(file: "builds/test-release-candidates/core/${productName}/${buildManifest.build.version}", bucket: "${ARTIFACT_PUBLIC_BUCKET_NAME}", path: "${artifactPath}/builds/${productName}/${fileLocation}")
```

Comment:
Is "core" some other convention? Can we change it to `min` or something like that?

Reply:
That's the original file system directory structure. @peterzhuamazon Is it ok if we change it to `min`?

Reply:
stay on core